### DISCUSSION TITLE
Whatsapp Conversations are not shown in IPED-SearchApp

### DISCUSSION THREAD

**Discussion by barraviera** [sepinf-inc/IPED#1591] [Q&A]
I processed an evidence using .UFDR image, however the Whatsapp conversations did not show up in IPED-SearchApp. The same evidence was processed using Cellebrite and the conversations appear normally. What can be happening?

**Comment by lfcnassif** [comment]
What version are you using? If you compiled IPED from source using master branch (unstable according to main project page), there was a bug introduced some days ago I fixed today, see #1586. If you are using 4.1.0 or a previous released version, we need the full processing log and ideally test data to reproduce the issue, you can share privately with me if it contains sensible data.

**Reply by barraviera** [reply]
Good afternoon. I used version 4.1.0. I noticed that audios, images, videos and others were processed, but the chats were not. After this adjustment #1586 is it necessary to change any configuration in my IPED?

**Reply by lfcnassif** [reply]
Version 4.1.0 is not affected by #1586. Please share your full processing log.

**Reply by lfcnassif** [reply]
Please reopen if you can provide the information below:
1. Full processing log;
2. If Databases were exported to UFDR report or not;
3. A sample UFDR report that reproduces the issue, if answer to (2) is yes.
